src/traces/indicator/defaults.js

```diff
@@ -63,7 +64,8 @@ function supplyDefaults(traceIn, traceOut, defaultColor, layout) {
         }
         coerce('delta.reference', traceOut.value);
         coerce('delta.relative');
-        coerce('delta.valueformat', traceOut.delta.relative ? '2%' : '.3s');
+        coerce('delta.valueformat');
+        if(!traceOut.delta.valueformat) traceOut.delta.valueformat = traceOut.delta.relative ? '+2%' : '+.3s';
```

Comment:
As you know, I'm not a fan of these `+` prefixes. They do make animations look better, but again not all `indicator` traces will be used with animations. 

I'd vote for bringing back `.3s` and adding some info in the `delta.tickformat` description about `+.3s` for users wanting more fixed-bbox delta tickers.

But, I'll let @plotly/plotly_js vote on this.

Reply:
Let me remove the `+` and check if animations are OK. I added more padding between `number` and `delta` so we might be fine!

Reply:
imo the dafualt should be set to blank; then under the hood '.3s' could be used (for the moment) until we figure out how to make blank work perfectly.

Reply:
I removed the `+` sign by default in commit b942832